csv2docx/csv2docx.py

```diff
@@ -20,11 +52,13 @@ def convert(data, template, delimiter=";"):
             return
 
         print("All fields are present in your csv. Generating Word docs ...")
+        path = create_path_if_not_exists(path)
 
-        for counter, row in enumerate(csvdict):
+        for row in csvdict:
             # Must create a new MailMerge for each file
             docx = MailMerge(template)
             single_document = {key : row[key] for key in docx_mergefields}
             docx.merge_templates([single_document], separator='page_break')
-            # TODO: write to user-defined subfolder
-            docx.write(f"{counter}.docx")
+            # Striping every name to remove extra spaces
+            filename = generate_name(row[custom_name].strip(), path)
```

Comment:
call `.strip()` inside `generate_name` as imho it's cleaner that way since we can more easily understand which params are being sent